src/presentation/react/core/components/sync-params-selector/SyncParamsSelector.tsx

```diff
@@ -256,6 +263,8 @@ const SyncParamsSelector: React.FC<SyncParamsSelectorProps> = ({ syncRule, onCha
                     value={
                         syncRule.type === "metadata" || syncRule.type === "deleted"
                             ? syncParams.importMode === "VALIDATE"
+                            : syncRule.type === "events"
+                            ? dataParams.importMode === "VALIDATE"
                             : dataParams.dryRun || false
```

Comment:
[subjective] Nested ternary conditionals are hard to read, maybe we can move it to a function/method (as if/elseif/else block)?